Sources/PackageDescription/CMakeLists.txt

```diff
@@ -1,75 +1,72 @@
 # This source file is part of the Swift.org open source project
 #
-# Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
 # Licensed under Apache License v2.0 with Runtime Library Exception
 #
 # See http://swift.org/LICENSE.txt for license information
 # See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 
-foreach(PACKAGE_DESCRIPTION_VERSION 4 4_2)
-    add_library(PD${PACKAGE_DESCRIPTION_VERSION}
-        BuildSettings.swift
-        LanguageStandardSettings.swift
-        PackageDescription.swift
-        PackageDependency.swift
-        PackageRequirement.swift
-        Product.swift
-        Resource.swift
-        SupportedPlatforms.swift
-        Target.swift
-        Version.swift
-        Version+StringLiteralConvertible.swift)
+add_library(PackageDescription
```

Comment:
While unwinding the loop I also made this conform to the 2-space indentation that we seem to be using for CMakefiles.